### PR TITLE
4365: search results action, sequence, table

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2020-04-02 at 1 10 03 PM

### CODE REVIEW

File: web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { state } from 'cerebral';
+
+/* sets props.practitioners on state.advancedSearchForm.results
```
Comment (matthopson) [inline web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js]: should be searchResults ?

File: web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {Function} providers.get cerebral get function
+ * @returns {object} contains the practitioners returned from the getPractitionersByNameInteractor use case
```
Comment (matthopson) [inline web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js]: Drop this

File: web-client/src/presenter/computeds/advancedSearchHelper.js [hunk 3]
```diff
@@ -45,21 +45,26 @@ export const advancedSearchHelper = (get, applicationContext) => {
...
+  const advancedSearchTab = get(state.advancedSearchTab);
   const currentPage = get(state.advancedSearchForm.currentPage);
   let result = {
     showPractitionerSearch: permissions.MANAGE_ATTORNEY_USERS,
     showStateSelect: countryType === COUNTRY_TYPES.DOMESTIC,
   };
   if (searchResults) {
-    const formattedSearchResults = searchResults.map(searchResult =>
-      formatSearchResultRecord(searchResult, { applicationContext }),
+    let formattedSearchResults = searchResults.slice(
```
Comment (matthopson) [inline web-client/src/presenter/computeds/advancedSearchHelper.js:55]: Shouldn't the pagination be handled in the response from ES?

File: web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  }) {
+    return (
+      <>
+        {advancedSearchHelper.showSearchResults && (
+          <>
+            <h1 className="margin-top-4">
+              ({advancedSearchHelper.searchResultsCount}) Results
+            </h1>
+
+            <table className="usa-table search-results docket-record responsive-table row-border-only">
```
Comment (matthopson) [inline web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx:25]: is this docket-record class doing anything useful?

File: web-client/src/presenter/computeds/advancedSearchHelper.js (hunk 3 above)
Comment (rachaelparris) [reply web-client/src/presenter/computeds/advancedSearchHelper.js:55]: That isn't how we do it for the case advanced search. We can discuss more if needed, but I think there's probably more overhead involved with doing multiple calls to the endpoint to load more pages than is necessary for the size of results that could be coming back here.

**Review by matthopson — APPROVED** [review]
I hate holding up in-progress stuff over piddly things, so feel free to address in a follow up PR if you want.

File: web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx (hunk 4 above)
Comment (rachaelparris) [reply web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx:25]: Surprisingly, yes. It has the table header styles. I think a renaming is probably in order. It looks like we use this class in a lot of places.

File: web-client/src/presenter/computeds/advancedSearchHelper.js (hunk 3 above)
Comment (matthopson) [reply web-client/src/presenter/computeds/advancedSearchHelper.js:55]: Fair enough :)

File: web-client/src/presenter/actions/AdvancedSearch/clearSearchResultsAction.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { state } from 'cerebral';
+
+/**
+ * clears the state.searchResults and sets the currentPage to 1
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.store the cerebral store
+ * @param {object} providers.props the cerebral props object
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/AdvancedSearch/clearSearchResultsAction.js]: nit: no props argument

File: web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js (hunk 2 above)
Comment (rachelschneiderman) [reply web-client/src/presenter/actions/AdvancedSearch/setPractitionerResultsAction.js]: nit: review all jsdocs?

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**